src/app/pages/i18n/i18n.component.html

```diff
@@ -0,0 +1,100 @@
+                </mat-toolbar-row>
+                <mat-toolbar-row>
+                    <small style="margin-top: -24px;" id="i18n">{{ 'I18N' | translate }}</small>
+                </mat-toolbar-row>
+            </mat-toolbar>
+            <mat-divider></mat-divider>
+            <div class="nav-wrapper">
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>home</mat-icon>
+                    <span>{{ 'MENU_ITEMS.HOME' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>folder</mat-icon>
+                    <span>{{ 'MENU_ITEMS.PROJECTS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>report</mat-icon>
+                    <span>{{ 'MENU_ITEMS.WARNINGS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>settings</mat-icon>
+                    <span>{{ 'MENU_ITEMS.SETTINGS' | translate }}</span>
+                </a>
+                <a mat-list-item (click)="toggleMenu()">
+                    <mat-icon matListIcon>help</mat-icon>
+                    <span>{{ 'MENU_ITEMS.HELP' | translate }}</span>
+                </a>
+            </div>
+        </mat-nav-list>
+    </mat-sidenav>
+    <mat-sidenav-content class="content">
+        <mat-toolbar color="primary">
+            <button *ngIf="isSmall" style="margin-left: -8px;" mat-icon-button (click)="toggleMenu()">
+                <mat-icon>menu</mat-icon>
+            </button>
+            <div>{{ 'I18N' | translate }}</div>
+            <pxb-spacer></pxb-spacer>
+            <button
+                type="button"
+                aria-label="Toggle sidenav"
+                mat-icon-button
+                (click)="openI18nMenu()"
+                style="margin-left: -4px; margin-right: 12px;"
+            >
+                <mat-icon aria-label="Side nav toggle icon">menu_open</mat-icon>
+            </button>
+        </mat-toolbar>
+
+        <mat-toolbar class="select-language">
+            <mat-form-field>
+                <mat-select
+                    #langeSelect
+                    (ngModelChange)="translate.use(langeSelect.value)"
+                    [(ngModel)]="selectedLanguage"
+                >
+                    <mat-option *ngFor="let lang of enabledLocales; let i = index" [value]="enabledLocales[i]">
+                        {{ 'LANGUAGES.' + lang | translate }}
+                    </mat-option>
+                </mat-select>
+            </mat-form-field>
+            <button mat-raised-button color="primary" style="margin: 20px;">
+                <mat-icon>shopping_cart</mat-icon> {{ 'ADD_TO_CART' | translate }}
+            </button>
+        </mat-toolbar>
+
+        <mat-nav-list>
```

Comment:
Can you make this mat-list instead?

Reply:
> do we want to adjust the snack-bar layout now since we have a side drawer? on a 1024 view snack-bar sitting center of entire app and not just the page...
> 
> ![image](https://user-images.githubusercontent.com/37150565/87589560-6427d880-c6b3-11ea-9db0-f790e989b9b1.png)

Yep you're right this should be centered in the nav content, not the entire screen.  It should also be a snackbar.

Reply:
You're going to have a tough time making this use the snackbar since it only supports a single action for the right side

Reply:
I don't mind this one being a mat-nav-list since the click behavior is functional (checks the checkbox)

Reply:
I got a snackbar working; it was a bit of headache trying to center it relative to the i18n content.